README.md

```diff
@@ -98,6 +98,14 @@ By using `params` option, `$fetch` adds params to URL by preserving params in re
 await $fetch('/movie?lang=en', { params: { id: 123 } })
 ```
 
+## ✔️ Adding headers
```

Comment:
This is a native functionality of fetch. What about using a lamp to indicate it is a tip? 

```suggestion
## 💡 Adding headers
```